frontend/src/components/KeyMeasureDiagnostic/WasteMeasure.vue

```diff
@@ -157,12 +157,71 @@
       :disabled="readonly"
       class="mt-6"
     ></v-textarea>
+
+    <v-divider v-if="showExpeSegment" class="mb-8"></v-divider>
+    <div v-if="showExpeSegment">
+      <p class="text-h6 font-weight-bold">
+        Expérimentation réservation de repas
+      </p>
+
+      <p class="body-2">
+        Vous souhaitez réduire le gaspillage alimentaire dans votre établissement et générer des économies :
+        <span class="font-weight-bold">la réservation de repas peut être une solution !</span>
+      </p>
+      <p class="body-2">
+        Pour évaluer ses effets sur le gaspillage alimentaire, la satisfaction de vos convives et le taux de
+        fréquentation de votre établissement, nous vous proposons de participer à une expérimentation prévue par la loi
+        Climat et résilience.
```

Comment:
```suggestion
        climat et résilience.
```
Utilisation de minuscules comme https://www.ecologie.gouv.fr/loi-climat-resilience